requirements/base.txt

```diff
@@ -1,3 +1,3 @@
-merlin-core>=0.2.0
-merlin-dataloader>=0.0.2
+merlin-core>=23.02.01
```

Comment:
The version numbers in PyPI don't have leading zeros. (E.g. 23.2.1) will this work like this?

Reply:
Yeah, seems like the install still works okay